UnityProject/Assets/Scripts/Items/Implants/Organs/Lungs.cs

```diff
@@ -107,10 +107,9 @@ public bool TryBreathing(IGasMixContainer node, float efficiency)
 			{
 				return false;
 			}
-
-			if (RelatedPart.HealthMaster.CirculatorySystem.BloodPool[RelatedPart.bloodType] == 0)
+			if (RelatedPart.HealthMaster.CirculatorySystem.BloodPool[RelatedPart.bloodType] == 0 || RelatedPart.HealthMaster.IsDead)
```

Comment:
```suggestion
			if (RelatedPart.HealthMaster.IsDead || RelatedPart.HealthMaster.CirculatorySystem.BloodPool[RelatedPart.bloodType] == 0)
```

Just invert that as its more likely to fail on death than no blood first. Very minor optimisation

Reply:
Might be better to move the death check to the beginning of the `ImplantPeriodicUpdate()`